packages/lib-classifier/src/store/feedback/helpers/generate-rules.js

```diff
@@ -0,0 +1,46 @@
+import _ from 'lodash'
```

Comment:
Should all `lodash` imports be scoped to just the methods used in the file?

Reply:
If these helpers are acting on MST nodes, we should look at scrapping lodash in favour of using MST's methods for maps and iterators.